common/djangoapps/enrollment/views.py

```diff
@@ -438,6 +438,8 @@ def post(self, request):
                     }
                 )
 
+            enrollment_attributes = request.DATA.get('enrollment_attributes')
```

Comment:
Since we're updating the API, we also need to update the documentation (the docstring for this view class).  FYI: @mhoeber